tasks/bd-stable.ts

```diff
@@ -15,6 +15,18 @@ export function load() {
       await lockBdusCrAt(hre, crRatio);
     });
 
+  task("bds:bxau:lockCollateralRatio")
+    .addPositionalParam("crRatio", "The desired collateral ratio")
+    .setAction(async ({ crRatio }, hre) => {
+      await lockBxauCrAt(hre, crRatio);
+    });
+
+  task("bds:bgbp:lockCollateralRatio")
+    .addPositionalParam("crRatio", "The desired collateral ratio")
+    .setAction(async ({ crRatio }, hre) => {
+      await lockBgbpCrAt(hre, crRatio);
+    });
+
```

Comment:
This is not scalable with multiple stables, maybe just have 1 task to lock and get a parameter which address/symbol?

Reply:
it means you'll have to search for the addresses and set them every time you execute the task. it's also more places for mistakes rather than having a dedicated task that already contains the correct names and addresses

Reply:
It's an NIT, IMO, so I'll let you decide.

You could also have the task get the symbol and search for the address.. Would be easier and more scalable.. We're looking at adding 6-8 more stables soon, this file is going to be horrible